### ISSUE TITLE
Tensorflow Lite New Converter does not allow to use inference_input_type and inference_output_type with V2 APIs

### ISSUE THREAD

**Issue by jinay1991** [tensorflow/tensorflow#36024]
**System information**
- OS Platform and Distribution (Linux, MacOS):
- TensorFlow installed from (official python wheel):
- TensorFlow version (v2.1.0):

```
    converter = tf.lite.TFLiteConverter.from_keras_model(model)

    converter.optimizations = [tf.lite.Optimize.OPTIMIZE_FOR_SIZE]
    converter.experimental_new_converter = True
    converter.experimental_new_quantizer = True
    converter.representative_dataset = representative_data_gen
    converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
    converter.inference_input_type = tf.uint8
    converter.inference_output_type = tf.uint8

    tflite_model = converter.convert()
```

Above code segment is expected to produce uint8 inference_input_type and inference_output_type, yet there is no uint8 conversion of the input layers and it ended up in identifying as floating_model when used in example [label_image.cc](https://github.com/tensorflow/tensorflow/blob/6ef62c6d2e90675eed0bb6ed10d8c5761ab365c1/tensorflow/lite/examples/label_image/label_image.cc#L226)

```
...
mobilenetv2_1.00_224/global_average_pooling2d/Mean <type 'numpy.int8'>
mobilenetv2_1.00_224/global_average_pooling2d/Mean/reduction_indices <type 'numpy.int32'>
mobilenetv2_1.00_224/out_relu/Relu <type 'numpy.int8'>
input_1 <type 'numpy.float32'>
Identity <type 'numpy.float32'>
```
Model used: [mobilenet_v2_1.0_224_quant.tgz](https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/mobilenet_v2_1.0_224_quant.tgz)

**Failure details**
If the conversion is successful, but the generated model is wrong,
state what is wrong:
- Unable to produce true uint8 input/output layers.

**Comment by ymodak** [comment]
I used [netron](https://github.com/lutzroeder/netron) to visualize your model. Please take a look.
<img width="791" alt="Screen Shot 2020-03-03 at 2 34 50 PM" src="https://user-images.githubusercontent.com/42785357/75826347-2fe2f980-5d5c-11ea-857e-e04abbeaace3.png">

**Comment by ymodak** [comment]
Closing this issue since it's resolved. Feel free to reopen if necessary. Thanks!

**Comment by yaronnaim** [comment]
No clear what was the solution.

**Comment by jinay1991** [comment]
This ha

> No clear what was the solution.

As I have noticed, this has been fixed and included in Release [v2.3.0](https://github.com/tensorflow/tensorflow/releases/tag/v2.3.0) -> Checkout Release Notes (tflite section)
